usecases/optimization_paper/optimization_workflow/Snakefile

```diff
@@ -362,13 +368,15 @@ rule approx_hydration_parameters:
         from lebedigital.demonstrator_scripts.dummy_hydration_parameters import dummy_hydration_parameters
          #merging contents of both dictionaries and individual variable inputs
 
+        # had to bypass this pint thing as it was not letting me use numpy.
```

Comment:
This sounds dangerous and is a great source for errors.
Can we move the "removing of pint units" to the script? There we can first make sure the right units are applied and then remove the units. Otherwise this will lead to problems. By using pint_object.magnitude, you can access the value of the object.